WalletWasabi.Fluent/ViewModels/Dialogs/Authorization/HardwareWalletAuthDialogViewModel.cs

```diff
@@ -55,4 +55,6 @@ protected override async Task<bool> Authorize()
 			return false;
 		}
 	}
+
+	protected override string AuthorizationFailedMessage => $"Authorization failed.{Environment.NewLine}Please, check that your device is connected and try again.";
```

Comment:
This is not 100% true. It can be several reasons why it failed. Something more general message would fit better.
```suggestion
	protected override string AuthorizationFailedMessage => $"Authorization failed.{Environment.NewLine}Please, check your device and try again.";
```